src/Microsoft.DotNet.Cli.Utils/ProjectContextExtensions.cs

```diff
@@ -63,5 +69,118 @@ public static string GetDefaultRootOutputPath(ProjectContext context, string cur
 
             return rootOutputPath;
         }
+        
+        internal static void MakeRunnable(this ProjectContext context, string outputPath, string configuration)
+        {
+            context
+                .ProjectFile
+                .Files
+                .GetCopyToOutputFiles()
+                .StructuredCopyTo(context.ProjectDirectory, outputPath)
+                .RemoveAttribute(FileAttributes.ReadOnly);
```

Comment:
This is the pre-existing behavior of this code. I've only factored it out so it's callable. We can look at simplifying it in a future PR.